src/utils/internal/getCallFrame.ts

```diff
@@ -2,25 +2,21 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  try {
-    const inspectionError = new Error()
-    inspectionError.name = 'Inspection Error'
-    throw inspectionError
-  } catch (error) {
-    const frames: string[] = error.stack.split('\n')
+  // Error.stack is available even in IE 10, no need for nullish coalesce
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack#browser_compatibility
+  const frames: string[] = new Error().stack!.split('\n')
 
-    // Get the first frame that doesn't reference the library's internal trace.
-    // Assume that frame is the invocation frame.
-    const declarationFrame = frames.slice(1).find((frame) => {
-      return !/(node_modules)?\/lib\/(umd|esm)\//.test(frame)
-    })
+  // Get the first frame that doesn't reference the library's internal trace.
+  // Assume that frame is the invocation frame.
+  const declarationFrame = frames.slice(1).find((frame) => {
+    return !/(node_modules)?\/lib\/(umd|esm)\//.test(frame)
+  })
 
-    if (!declarationFrame) {
-      return
-    }
-
-    // Extract file reference from the stack frame.
-    const [, declarationPath] = declarationFrame.match(/\((.+?)\)$/) || []
-    return declarationPath
+  if (!declarationFrame) {
+    return
   }
+
+  // Extract file reference from the stack frame.
+  const [, declarationPath] = declarationFrame.match(/\((.+?)\)$/) || []
```

Comment:
Did you have a chance to examine whether Firefox stack traces match this format? 

```
  at (<FILE_PATH>)
```

Reply:
nope. do you have any instructions how to execute this code in browser? I can click the `here` button for CodeSandbox, but what can I do to execute `getCallFrame` plz???

Reply:
Sure, I'll share the instructions below. 

## Executing in a browser

1. Clone the repo and install its dependencies.
2. Run a single integration test in a browser. You'd need a test that exposes the `msw` object globally. For example `yarn test:focused test/msw-api/setup-worker/printHandlers.mocks.ts`
3. Open the spawned URL in Firefox.
4. Open DevTools, switch to "Console".
5. Run `window.msw.worker.printHandlers()`.

That should print the list of handlers into the console. As a part of each message, there will be a call frame to the handler's definition.

Reply:
Screenshot from output in Chrome (left), Safari (middle), and Firefox (right),, on macOS 10.14:

<img width="1680" alt="Screenshot 2021-01-29 at 19 07 03" src="https://user-images.githubusercontent.com/1087670/106311729-e483e480-6265-11eb-894e-a8d1a6693243.png">

Looks like the "Declaration" property is `undefined` => I will try to investigate.

Reply:
OK, I pushed a commit that will make Firefox output the same Declaration as Chrome, and Safari to output the best it can.

The unit test is necessary for TDD. As I mentioned in issue #562, I was not able to run the integration / focused test on my Windows => if someone else can add unit tests from more supported browsers or from a Deno server, feel free to tag me to help with improving the implementation.

Reply:
arghh@!$# I need to fix `setup-server/printHandlers.test.ts` .. looks like Node has a slightly different stack than Chrome after all